src/pages/MonthlyGood/MonthlyGoodSetupPageVariant.vue

```diff
@@ -203,6 +203,12 @@
 </template>
 
 <script>
+/**
```

Comment:
Should we delete this template and the wrapper template that decides which setup form?

Reply:
hmmm. I know the payment section of the variant is now out of date so we def want to prevent this from being turned on at all. That said, I don't know if we want to delete it outright just yet as it may have some useful code for the new designs...but we can always look back at a release branch to anything useful... so yea I'm ok removing it and cleaning up the exp codes.

Reply:
I'll remove it in a followup PR. In the SUBS backlog, there are items for a MG Landing page redesign, as well as exploring making monthly good signup a single page vs landing > setup.